arc/src/arc_curie_pme.c

```diff
@@ -33,12 +34,13 @@ void curie_pme_configure(u16_t global_context,
 }
 
 // clear all commits in the network and make it ready to learn
-void curie_pme_forget(void) { reg_write16(FORGET_NCOUNT, 0); }
+void curie_pme_forget(void)
+{
+    reg_write16(FORGET_NCOUNT, 0);
+}
 
 // mark --learn and classify--
-u16_t curie_pme_learn(u8_t *pattern_vector,
-                      s32_t vector_length,
-                      u16_t category)
+u16_t curie_pme_learn(u8_t *pattern_vector, s32_t vector_length, u16_t category)
```

Comment:
Like I said previously, if it fits in the same line, it will put it as 1 line, but if it doesn't fit, then it defaults to break each argument into separate line, but I am okay with fit everything as you can on a per line basis.